### PR TITLE
Added ability to ask questions about the url

### PR DESCRIPTION
**Pull request by Tom-Hudson**
Addresses feature request #111

### CODE REVIEW

**Comment by Tom-Hudson** [comment]
As mentoned in #111, my changes wouldn't have caused the build to fail!

File: packages/serenity-js/spec/cookbook/content.recipe.ts [hunk 1]
```diff
@@ -49,6 +49,11 @@ describe ('When demonstrating the usage of a HTML page, a test scenario', functi
         expect(james.toSee(Website.title()))
             .eventually.equal('Serenity/JS Cookbook'));
 
+    it ('can read the url of the website', () =>
+
+        expect(james.toSee(Website.url()))
+            .eventually.equal('Serenity/JS Cookbook'));
```
Comment (jan-molak) [inline packages/serenity-js/spec/cookbook/content.recipe.ts]: The assertion doesn't seem correct?

**Comment by jan-molak** [comment]
Yes, I think there's an issue with the PR build, I need to have a look into it. Sorry for the inconvenience.

**Comment by Tom-Hudson** [comment]
I have now addressed the review comments. My bad, total copy and paste fail!

**Comment by jan-molak** [comment]
Awesome, thanks for your contribution!

**Pull request merged**